js/theme.js

```diff
@@ -0,0 +1,74 @@
+const themeToggleBtn = document.querySelector(".theme-toggle-btn");
+const body = document.querySelector("body");
+
+const rootElement = document.querySelector(":root");
+
+const darkThemeColors = () => {
+    rootElement.style.setProperty("--theme-bg", "#141414");
+    rootElement.style.setProperty("--theme-text-color", "#bbbbbb");
+    rootElement.style.setProperty("--navbar-bg", "#0e0e0e")
+    rootElement.style.setProperty("--primary","#1c96ff");
+    rootElement.style.setProperty("--primary-hover","#133e64");
+    rootElement.style.setProperty("--info-hover","#cbe2ff");
+    rootElement.style.setProperty("--border-color","#646464");
+    rootElement.style.setProperty("--shadow","#3c3c3c");
+    rootElement.style.setProperty("--card-button-bg","#080808");
+    rootElement.style.setProperty("--code-snippet", "#0c0c0c");
+    rootElement.style.setProperty("--overlay", "rgba(56, 56, 56,0.60)")
+};
+
+const lightThemeColors = () => {
+    rootElement.style.setProperty("--theme-bg", "#ffffff");
+    rootElement.style.setProperty("--theme-text-color", "#313131");
+    rootElement.style.setProperty("--navbar-bg","#ffffff")
+    rootElement.style.setProperty("--primary","#19456b");
+    rootElement.style.setProperty("--primary-hover","#133e64");
+    rootElement.style.setProperty("--secondary","#16c79a");
+    rootElement.style.setProperty("--secondary-hover","#0eb98f");
```

Comment:
This much of setup was unnecessary for toggling themes. A better way would be to define the whole theme already.

```
.light-theme{
    --primary: #fff;
}
.dark-theme{
    --primary:#000;
}
```
Now toggle the class on the body depending on the button click.

Reply:
I tried that. But the dark theme colors are not getting applied. Need more time to debug. So for time being, I am moving forward with this approach. 
Thanks for the review. :)